tests/chat/test_sequences.py

```
@@ -0,0 +1,211 @@
+"""
+Test suite for chat sequence validation
+Tests initialization and exit procedures
+"""
+
+import pytest
+from datetime import datetime
+from unittest.mock import Mock, patch
```

Ruff (0.8.2)

`unittest.mock.Mock` imported but unused

Remove unused import

(F401)